### PR TITLE
Add documentation for 5 more checks

### PR DESCRIPTION
**Pull request by dcastrowa**
### Description:

Add check documentation for the checks listed below:
- LongSegmentCheck
- OverlappingEdgeCheck
- ShortSegmentCheck
- RoadLinkCheck
- ShortSegmentCheck

### CODE REVIEW

File: docs/available_checks.md [hunk 1]
```diff
@@ -23,23 +23,23 @@ This document is a list of tables with a description and link to documentation f
...
+| [LongSegmentCheck](checks/longSegmentCheck.md) | This check identifies long segments/edges (length is more than minimumLength). |
 | [MalformedRoundaboutCheck](checks/malformedRoundaboutCheck.md) | The purpose of this check is to identify roundabouts mapped in the opposite direction of traffic. The check takes into consideration countries with both left-side and right-side traffic. There are three types of map errors that will be flagged by this check: 1. Wrong-way-roundabouts, 2. Multi-directional roundabouts, and 3. Roundabouts with poor geometry. |
-| OverlappingEdgeCheck | The purpose of this check is to identify edges that share the same two consecutive geometry points. |
-| RoadLinkCheck |  Verifies that one end or the other of an edge is a fork to/from a road of the same class, that is not a _link. |
+| [OverlappingEdgeCheck](checks/overlappingEdgeCheck.md) | The purpose of this check is to identify edges that share the same two consecutive geometry points. |
+| [RoadLinkCheck](checks/roadLinkCheck.md) |  Verifies that one end or the other of an edge is a fork to/from a road of the same class, that is not a _link. |
```
Comment (mattmanley) [inline docs/available_checks.md:29]: @dcastrowa Is this supposed to say "_link"?

**Review by ameliaewatts — APPROVED** [review]
This looks good from a non-technical stand point. 👍

File: docs/available_checks.md (hunk 1 above)
Comment (dcastrowa) [reply docs/available_checks.md:29]: I believe that what it says originally but I think it should be just say "link" without the underscore.

**Review by mattmanley — APPROVED** [review]
LGTM! Thanks!

**Review by matthieun — APPROVED** [review]
Nice doc!

**Pull request merged**